coalib/collecting/Collectors.py

```diff
@@ -110,6 +110,9 @@ def icollect_bears(bear_dirs, bear_globs, kinds, log_printer):
     """
     for bear_dir, dir_glob in filter(lambda x: os.path.isdir(x[0]),
                                      icollect(bear_dirs)):
+        # Since we get a real directory here and since we
+        # pass this later to iglob, we need to escape this.
+        bear_dir = glob_escape(bear_dir)
```

Comment:
So basically break this commit into three, right?

- First with this bug fix
- The `coala_delete_orig.py` one
- Then the others (as they are all file collection using `glob_list` instead of `path_list`)